app/public/js/home.js

```diff
@@ -227,4 +228,38 @@ function bindCategoryLinksToKeywordInput()
 	updateCategoryLinksOffKeywords();
 }
 
-$(document).ready(bindCategoryLinksToKeywordInput);
\ No newline at end of file
+$(document).ready(bindCategoryLinksToKeywordInput);
+
+
+function blinkMap(map1)
+{
+	fadeOut(map1,0.5)
```

Comment:
I read about the jQuery solution and people saw it on stackoverflow and said that it was noticeably slower, especially on mobile.

Reply:
It supports an optional time parameter so if you want to speed it up.  The default is 400ms so anything less is faster.

```$(map1).fadeOut(200, function(){ $(map1).fadeIn(200); });```

Reply:
I meant it was less efficient so it might be laggy on mobile. I'll change it to the jQuery option and if we notice problems on mobile at a later date we can change it back